pkg/image/api/helper.go

```diff
@@ -128,6 +128,18 @@ func (r DockerImageReference) AsRepository() DockerImageReference {
 	return r
 }
 
+// DaemonMinimal clears defaults that Docker assumes.
```

Comment:
by breaking them if they do?  what's the motivation for doing this minimization?